styles/main.scss

```diff
@@ -46,6 +48,21 @@ $font-sizes: (
 $font-family-base: "Proxima Nova", "Open Sans", "sans-serif";
 $font-family-secondary: "adelle", "Aleo", "serif";
 
+// From design library, please prefer media-breakpoint-up/down/only/between to target certain breakpoints
+// See also: https://getbootstrap.com/docs/5.2/layout/breakpoints/#media-queries
+// xs + sm: 365px (mobile)
+// md     : 768px (tablet)
+// lg + xl: 1280px (desktop)
+// xxl    : 1440px (wide screen desktop)
+$grid-breakpoints: (
+  xs: 0,
+  sm: 365px,
+  md: 768px,
+  lg: 992px,
+  xl: 1280px,
+  xxl: 1440px,
+);
```

Comment:
Simon, so if I'm not mistaken, I think these were the defaults that we've been using so far: https://getbootstrap.com/docs/5.0/layout/breakpoints/
So the biggest difference would be the xs and sm changes, but from md and up is the same I think. So maybe we shouldn't touch anything we previously made md and up?

Reply:
Well technically we are going from 6 sizes to 4, (or 5 if you want to discount xs as it's 0), yes from md and up the difference is minimal, but the design library on figma is our source of truth (where possible), not bootstrap
since we are going from

- xs: 0 = 0 (no corresponding size)
- sm: 576 -> 365 (mobile)
- md: 768 = 768 (tablet)
- lg: 992 = 992 (no corresponding size, avoid use)
- xl: 1200 -> 1280 (desktop)
- xxl: 1400 -> 1440 (wide screen desktop)

If I thought it was safe to do so, I think I would advocate for removing `lg` actually.
I think we can make a case to Lydia to have the xl, xxl new sizes match bootstrap, but either way I think we should stop using lg